tensorboard/plugins/interactive_inference/witwidget/notebook/base.py

```diff
@@ -462,7 +462,7 @@ def create_selection_callback(self, examples, max_examples):
     def handle_selection(selected):
       def extract_values(feat):
         if feat.HasField('bytes_list'):
-          return feat.bytes_list.value
+          return [v.decode('utf-8') for v in feat.bytes_list.value]
```

Comment:
just want to verify, does this fix work in both py2 and py3

Reply:
Hi, just tested on python2 colab. It also works with python2.